src/tlo/methods/breast_cancer.py

```diff
@@ -575,6 +575,7 @@ class BreastCancerMainPollingEvent(RegularEvent, PopulationScopeEventMixin):
     def __init__(self, module):
         super().__init__(module, frequency=DateOffset(months=1))
         # scheduled to run every 3 months: do not change as this is hard-wired into the values of all the parameters.
+        # TODO: Is it? There is 1 month as the frequency, isn't it?
```

Comment:
Good spot. Please update the comment!

Reply:
Hi @EvaJanouskova Thanks - I've looked back at this.  It looks like you are right that we moved over to a 1 month update.  So the comment can be updated or removed.   I also see that the write-up needs updating so I will try to do this when I can.